protoV2/tcAbacus/danielvladco/protobuf/graphql.stub.proto

```
@@ -0,0 +1,62 @@
+syntax = "proto2";
+package danielvladco.protobuf.graphql;
```

Buf (1.66.1)

[error] Files with package "danielvladco.protobuf.graphql" must be within a directory "danielvladco/protobuf/graphql" relative to root but were in directory "protoV2/tcAbacus".

(PACKAGE_DIRECTORY_MATCH)